src/enums.h

```diff
@@ -59,6 +59,8 @@ enum itemAttrTypes : uint32_t {
 	ITEM_ATTRIBUTE_CHARGES = 1 << 20,
 	ITEM_ATTRIBUTE_FLUIDTYPE = 1 << 21,
 	ITEM_ATTRIBUTE_DOORID = 1 << 22,
+
+	ITEM_ATTRIBUTE_CUSTOM = static_cast<uint32_t>(1) << 31
```

Comment:
You don't need to cast to uint32_t, 1 << 31 fits into `int`. Alternatively you can use `1u << 32`